packages/itwin/tree-widget/src/types.ts

```diff
@@ -5,34 +5,46 @@
 
 import type { SpatialContainmentTreeProps, StagePanelLocation, StagePanelSection } from "@itwin/appui-react";
 import type { SelectableContentDefinition } from "@itwin/components-react";
-import type { IModelConnection } from "@itwin/core-frontend";
+import type { IModelConnection, ScreenViewport } from "@itwin/core-frontend";
 import type { CategoryTreeProps } from "./components/trees/category-tree/CategoriesTree";
 import type { ModelsTreeProps } from "./components/trees/models-tree/ModelsTree";
+import type { Category, ModelsTreeComponentNamespace } from "./tree-widget-react";
+
+export interface TreeHeaderButtonProps {
+  viewport: ScreenViewport;
+}
+
+export interface ModelsTreeHeaderButtonProps extends TreeHeaderButtonProps {
+  models?: ModelsTreeComponentNamespace.TreeViewModelInfo[];
```

Comment:
I think there is not need to put `TreeViewModelInfo` under namespace. Also it probebly could be renamed to `ModelInfo`?

Reply:
fixed